### PR TITLE
Remove embeddable and replace InvoicingChannel to Channel form Core

### PR DESCRIPTION
**Pull request by Tomanhez**

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### UPGRADE FROM 0.9 TO 0.10.0
+
+1. Removed ``InvoicingChannel`` and replaced by ``Sylius\Component\Core\Model\ChannelInterface``.
+
+2. Replaced  ``InvoiceShopBillingData`` value object by entitie with ``InvoiceShopBillingDataInterface`` interface.
```
Comment (pamil) [inline UPGRADE.md]:
```suggestion
2. Replaced  `InvoiceShopBillingData` value object by entity with `InvoiceShopBillingDataInterface` interface.
```

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+### UPGRADE FROM 0.9 TO 0.10.0
+
+1. Removed ``InvoicingChannel`` and replaced by ``Sylius\Component\Core\Model\ChannelInterface``.
```
Comment (pamil) [inline UPGRADE.md]:
```suggestion
1. Removed `InvoicingChannel` and replaced by `Sylius\Component\Core\Model\ChannelInterface`.
```

File: src/Resources/config/doctrine/Invoice.orm.xml [hunk 3]
```diff
@@ -30,9 +30,16 @@
             </cascade>
         </one-to-many>
 
-        <embedded name="channel" class="Sylius\InvoicingPlugin\Entity\InvoiceChannel" />
+        <many-to-one field="channel" target-entity="Sylius\Component\Core\Model\ChannelInterface">
+            <join-column name="channel_id" referenced-column-name="id" nullable="true" />
```
Comment (pamil) [inline src/Resources/config/doctrine/Invoice.orm.xml]:
```suggestion
            <join-column name="channel_id" referenced-column-name="id" nullable="false" />
```

Comment (GSadee) [reply src/Resources/config/doctrine/Invoice.orm.xml]: Or even
```suggestion
            <join-column name="channel_id" />
```
as the rest values are the default ones

File: UPGRADE.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+### UPGRADE FROM 0.9 TO 0.10.0
+
+1. Removed `InvoicingChannel` and replaced by `Sylius\Component\Core\Model\ChannelInterface`.
+
+2. Replaced  `InvoiceShopBillingData` value object by entitie with `InvoiceShopBillingDataInterface` interface.
```
Comment (GSadee) [inline UPGRADE.md]:
```suggestion
2. Replaced  `InvoiceShopBillingData` value object by entity with `InvoiceShopBillingDataInterface` interface.
```

**Comment by GSadee** [comment]
Thank you, Tomasz! :tada:

**Pull request merged**